src/olympia/blocklist/tests/test_cron.py

```diff
@@ -398,9 +696,10 @@ def test_get_last_generation_time(self):
         assert get_last_generation_time() == 1
 
     def test_get_base_generation_time(self):
-        assert get_base_generation_time() is None
-        set_config(MLBF_BASE_ID_CONFIG_KEY(BlockType.BLOCKED, compat=True), 1)
-        assert get_base_generation_time() == 1
+        for block_type in BlockType:
+            assert get_base_generation_time(block_type) is None
+            set_config(MLBF_BASE_ID_CONFIG_KEY(block_type, compat=True), 1)
+            assert get_base_generation_time(block_type) == 1
```

Comment:
I forgot to comment on this before but I tend to avoid values such as `0` or `1` than can easily be transformed into something else (e.g. a boolean). Not a big deal.

```suggestion
            set_config(MLBF_BASE_ID_CONFIG_KEY(block_type, compat=True), 123)
            assert get_base_generation_time(block_type) == 123
```